Image publishing github workflow is broken

```
16.74 
16.74 https://nextjs.org/docs/messages/module-not-found
16.74 
16.74 ./src/components/Contribute/Skill/index.tsx
16.74 Module not found: Can't resolve 'js-yaml'
16.74 
16.74 https://nextjs.org/docs/messages/module-not-found
16.74 
16.75 
16.75 > Build failed because of webpack errors
```

List of recently failed jobs 
https://github.com/instructlab/ui/actions/workflows/images.yml

--- a/package.json
+++ b/package.json
@@ -36,12 +36,12 @@
     "react-dom": "18.3.1",
     "react-query": "^3.39.3",
     "typescript": "5.4.5",
-    "winston": "^3.13.0"
+    "winston": "^3.13.0",
+    "@types/js-yaml": "^4.0.9"
   },
   "devDependencies": {
     "@next/eslint-plugin-next": "^14.2.3",
     "@redhat-cloud-services/eslint-config-redhat-cloud-services": "^2.0.4",
-    "@types/js-yaml": "^4.0.9",
     "@types/styled-components": "^5.1.34",
     "@types/uuid": "^9.0.8",
     "@typescript-eslint/eslint-plugin": "^7.13.0",

